### ISSUE TITLE
[FEATURE]: Patch meta information of `torch.nn.Dropout`

### ISSUE THREAD

**Issue by Cypher30** [hpcaitech/ColossalAI#2631]
This is a part of issue #2628, we will patch meta information of `torch.nn.Dropout`
